app/src/main/java/com/kickstarter/ui/adapters/DiscoveryPagerAdapter.java

```diff
@@ -40,8 +38,11 @@ public void setPrimaryItem(final @NonNull ViewGroup container, final int positio
   }
 
   @Override
-  public @NonNull Fragment getItem(final int position) {
-    return DiscoveryFragment.newInstance(position);
+  @NonNull
```

Comment:
in general, we put our nullability annotations right in front of the type which it is annotating, i.e. 

```java
public @NonNull DiscoveryFragment
``` 

i'll update our styleguide to include this, since it's something we discussed a while ago as a team but neglected to include in the styleguide!